### PR TITLE
adding redhat support

### PR DESCRIPTION
**Pull request by justone**

### CODE REVIEW

File: tasks/install-redhat.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  become: true
+  shell: "logdna-agent -k {{ logdna_secret_key }}"
+
+- name: Configure | directory log location
+  become: true
+  shell: "logdna-agent -d {{ logdna_logging_directory }}"
+
+- name: Configure | service | logdna-agent
+  become: true
+  shell: "chkconfig logdna-agent on"
```
Comment (pdufault) [inline tasks/install-redhat.yml]: Can we use the ansible service module here so this is idempotent? 
service:
  name: logdna-agent
  enabled: yes

**Comment by pdufault** [comment]
Thanks for your contribution!

I've added a note on how the service is enabled, and the meta/main.yml file will need updating to reflect that RedHat is a supported distro on Ansible Galaxy.

Once those small changes are resolved this can be merged in.

**Comment by justone** [comment]
@pdufault I switched to the service module and updated the metadata.  Could you take another look?

**Comment by aaronpederson** [comment]
Thank you for your contribution, we very much appreciate it.

**Pull request merged**